examples/pix2pose/eval.py

```diff
@@ -0,0 +1,89 @@
+import numpy as np
+from scipy import spatial
+from scipy.spatial import distance
+from paz.backend.groups.quaternion import quaternion_to_rotation_matrix
+
+
+def get_translation(pose6D):
+    """It outputs the predicted Translation from the detected object.
+
+      # Arguments
+          pose6D: The 6d pose
+
+      # Returns
+          Return Translation vector
```

Comment:
Change here for "Translation vector array (3)"

Reply:
resolved